sonar/modules/documents/mappings/v6/documents/document-v1.0.0.json

```diff
@@ -319,7 +319,15 @@
           }
         },
         "usageAndAccessPolicy": {
-          "type": "text"
+          "type": "object",
```

Comment:
I do not understand why this property name is different between document and deposit...

Reply:
Because the deposit schema has been done a long time before adding the `usageAndAccessPolicy ` property in document schemas. And I don't have the time to refactor it.